packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -23,6 +30,25 @@ import {
 } from '@backstage/plugin-catalog';
 import { Entity } from '@backstage/catalog-model';
 import { Grid } from '@material-ui/core';
+import { WarningPanel } from '@backstage/core';
+
+const CICDSwitcher = ({ entity }: { entity: Entity }) => {
```

Comment:
🚀 

Unrelated question: We sometimes have multiple build servers. Ex: A hybrid app is built and deployed as a web-preview on a Jenkins, but the iOS and Android versions are built on bitrise. I'm curious whether the routing would somehow support two sub-routers.

Reply:
We chose this "write your own middle-layer glue" solution because we wanted to be able to support that kind of flexibility - yes, you could wire this together basically in any way you prefer. Either as two separate tabs, or as one generic tab inside of which is a sidebar page with two different build systems, or as one generic tab inside of which you have one jenkins card and one bitrise card, one above the other.